frontend/src/main/scala/akka/viz/frontend/FrontendUtil.scala

```diff
@@ -1,6 +1,11 @@
 package akka.viz.frontend
 
-import org.scalajs.dom._
+import org.scalajs.dom.Element
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/lustefaniak/akka-viz/file/2004142120/issues/source?bid=3121282&fileBranchId=3122179#l3)